xml/docu_styleguide.structure.xml

```diff
@@ -492,10 +492,10 @@ in the directory &lt;filename&gt;/etc/sysconfig&lt;/filename&gt;.</screen>
     <tag class="emptytag">screen</tag>
     element, always prefix them with a prompt marked up this way:
    </para>
-<screen>&lt;prompt&gt;tux &amp;gt; &lt;/prompt&gt;&lt;command&gt;ls&lt;/command&gt;</screen>
+<screen>&lt;prompt&gt;&amp;gt; &lt;/prompt&gt;&lt;command&gt;ls&lt;/command&gt;</screen>
    <para>
-    To avoid making prompts longer than necessary, do not include a host name
-    or a path in them, unless this is vital to understanding.
+    To avoid making prompts longer than necessary, do not include paths, user
+    or host names, unless this is vital to understanding.
```

Comment:
[Who cares about an Oxford comma?](https://www.youtube.com/watch?v=P_i1xk07o4g)
```suggestion
    To avoid making prompts longer than necessary, do not include paths, user,
    or host names, unless this is vital to understanding.
```